c2corg_ui/static/js/maincontroller.js

```diff
@@ -92,3 +97,9 @@ app.MainController.prototype.isPath = function(path) {
 app.MainController.prototype.animateHeaderIcon = function(e) {
   app.utils.animateHeaderIcon(e);
 };
+/**
+ * @export
+ */
+app.MainController.prototype.updateMapResize = function() {
+  this.scope_ .$root.$emit('updateMapResize');
```

Comment:
The event name is not very consistent. What about ``resizeMap`` instead?
The function name is not very consistent either. What about simply ``resizeMap``?

In addition if find it a bit weird to add such a function to the main controller. This function is not used in any page. What about adding it to the advanced search controller instead? https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/search/advancedsearch.js

Just a silly question: isn't it possible to call ``mapCtrl`` in the toggle buttons? If yes, do we really need such a function?